### PR TITLE
TCD-2585

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by LoryApiFortress** [comment]
@aliciakrebs Can you please review? Could it work in this way? (TCD-2585) For now, I changed only "Assert compares" if you think it is ok, I'll proceed with all the other assertions. Let me know your thought. Thanks

File: docs/api-testing/composer/assertion-components.md [hunk 1]
```diff
@@ -71,11 +98,11 @@ This assertion is used to check if the element described by the expression conta
...
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
 * __Value__: The value we want to compare the expression to.
 * __Type__: The type of the value. **Auto** means the engine will try to identify the type of the value.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specifies, when the assertion fails, whether it should be considered an **error** or just a **warning**.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:108]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 2]
```diff
@@ -121,11 +148,11 @@ This assertion is used to check if the element value described by the expression
...
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
 * __Value__: The value we want to compare the expression to.
 * __Type__: The type of the value. **Auto** means the engine will try to identify the type of the value.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:158]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 3]
```diff
@@ -121,11 +148,11 @@ This assertion is used to check if the element value described by the expression
 | Stop test if fails | Yes, No | No |
 
 
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:151]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 4]
```diff
@@ -168,9 +195,9 @@ This assertion is used to check if the element described by the expression exist
 | Stop test if fails | Yes, No | No |
 
 
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:198]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 5]
```diff
@@ -209,8 +236,8 @@ This assertion is used to check if the element value described by the expression
 
 * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
 * __Value__: The value we want to compare the expression to.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:243]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 6]
```diff
@@ -249,10 +276,10 @@ This assertion is used to check if the element described by the expression match
...
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
 * __Value__: The value we want to compare the expression to.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:285]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 7]
```diff
@@ -313,8 +340,8 @@ This assertion is used to check if the value of the element defined by the expre
     * _email_: checks if field is a valid email format;
     * _map_: checks if field is a map type;
     * _array_: checks if the field is an array.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:347]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 8]
```diff
@@ -353,10 +380,10 @@ This assertion is used to check if the element value described by the expression
...
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
 * __Value__: The value we want to compare the expression to.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:389]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 9]
```diff
@@ -404,8 +431,8 @@ This assertion is used to check if the element value described by the expression
     * _country codes_: checks if the field contains a valid country code (i.e., 'US', 'FR', 'DK');
     * _currency codes_: checks if the fields is a valid currency (i.e., 'USD', 'EUR');
 * __Regex value__: Specify the regular expression you want to use for checking the expression.
-* __Comment__: Add comment messages in the form of a string data type.
-* __Level__: Specify if the assertion fails whether it should be considered an ‘error’ or just a ‘warning’.
+* __Comment__: See [Comment](#comment).
+* __Level__: See [Level](#level).
 * __Modifier__: The assertion is considered verified if it does not pass.
 * __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.
 * __Stop test if fails__: The test will be immediately stopped if the assertion fails.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:438]:
```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

File: docs/api-testing/composer/assertion-components.md [hunk 10]
```diff
@@ -209,8 +236,8 @@ This assertion is used to check if the element value described by the expression
 
 * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:237]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 11]
```diff
@@ -249,10 +276,10 @@ This assertion is used to check if the element described by the expression match
 | Stop test if fails | Yes, No | No |
 
 
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:279]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 12]
```diff
@@ -303,7 +330,7 @@ This assertion is used to check if the value of the element defined by the expre
...
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:333]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 13]
```diff
@@ -353,10 +380,10 @@ This assertion is used to check if the element value described by the expression
 
 
 
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:383]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 14]
```diff
@@ -395,7 +422,7 @@ This assertion is used to check if the element value described by the expression
 | Stop test if fails | Yes, No | No |
 
 
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:425]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

File: docs/api-testing/composer/assertion-components.md [hunk 15]
```diff
@@ -435,7 +462,7 @@ This assertion is used to validate a JSON schema, based on the provided schema d
 | JsonSchema | JSON schema definition | Yes |
 | Comment | String | No |
 
-* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](https://apifortress.com/doc/expression/) for more details.
+* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
```
Comment (aliciakrebs) [inline docs/api-testing/composer/assertion-components.md:465]:
```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

**Comment by aliciakrebs** [comment]
This note regarding  Level  needs to be removed from the parameters tables. It is already mentioned in the referenced section.  If you want to keep it in the parameters tables, it should be moved under **Level**.  

:::note
A **warning** will not trigger alerts (such as email or text messages).
:::

**Comment by LoryApiFortress** [comment]
Thanks @aliciakrebs but as I said in my previous comment 
`For now, I changed **only** "Assert compares" if you think it is ok, I'll proceed with all the other assertions.` so, of course, all the other sections do not have the updates. I did this because I didn't want to waste time changing all the page if it turns out is not the correct way to change it.

**Comment by LoryApiFortress** [comment]
@aliciakrebs **ALL page** is now changed. Please review. Thanks

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**